dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/java/de/digitalcollections/cudami/server/backend/impl/jdbi/identifiable/IdentifiableRepositoryImpl.java

```diff
@@ -753,9 +753,6 @@ public I retrieveOne(
                 + (StringUtils.hasText(sqlAdditionalJoins)
                     ? " %s".formatted(sqlAdditionalJoins)
                     : "")
-                + (StringUtils.hasText(getSqlSelectReducedFieldsJoins())
-                    ? " %s".formatted(getSqlSelectReducedFieldsJoins())
-                    : "")
```

Comment:
Nein, das war schon genau so gedacht:  Passend zu den Feldern sollen die Joins in den jeweiligen Methoden "hinterlegt" werden.

Reply:
ok